posthog/api/insight.py

```diff
@@ -283,6 +285,30 @@ def calculate_funnel(self, request: request.Request) -> Dict[str, Any]:
         self._refresh_dashboard(request=request)
         return {"result": result}
 
+    # ******************************************
```

Comment:
Since Clickhouse only, this should go in `ee/clickhouse/views/insight.py` (and that in turn has the signature

`FunnelCorrelation(filter, team).run()`

Reply:
Right. It seems confusing to have a different URL structure depending on the backend but will update

Reply:
writing in `posthog/api/insight` or `ee/clickhouse/views/insight.py` wouldn't change the URL structure: that remains the same, it's just about which URLs are permitted per backend.

You'll notice that `views/insight.py` inherits from `api/insight`, and basically overrides some methods, without changing the URL structure.